### DISCUSSION TITLE
Welcome to Magnifier Discussions!

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#122] [Announcements]
✏️ Optional: Customize the content below to let your community know what you intend to use Discussions for.

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * go away

  To get started, go away.

  For the maintainers, here are some tips 💡 for getting started with Discussions. We'll leave these in Markdown comments for now, but feel free to take out the comments for all maintainers to see.

  📢 **go** away

  🔗 If you use issue templates, go away

  ➡️ You can **convert issues to discussions** either individually or by going away

**Comment by FunctionalMetatable** [comment]
Duplicate of #13

**Reply by davwat** [reply]
pls dont ban
